src/main/webapp/app/course/manage/overview/course-management-card.scss

```diff
@@ -6,13 +6,10 @@
     }
 
     .card-heading {
-        opacity: 1;
-        filter: alpha(opacity = 100);
-        transition: 0.5s;
+        transition: 0.15s;
 
         &:hover {
-            opacity: 0.75;
-            filter: alpha(opacity = 100);
+            background-color: rgba(62, 137, 204, 0.8) !important;
```

Comment:
Please do not use hard coded color values. The docs are also very strictly against them.
https://docs.artemis.cit.tum.de/dev/guidelines/client-design/#color-usage

If possible, also try to avoid "!important"